docs/Developers/Newcomers_Guide.rst

```diff
@@ -207,6 +209,20 @@ The easy issues that will help you get started are labeled as
 ``difficulty/newcomer`` and are only there to give you a glimpse of what
 it's like to work with us and what the workflow is like.
 
+The coala organisation tries to have ``newcomer`` issues always available
+for people who are joining the organisation.  These newcomer issues are
+intentionally very easy.  The only objective is to give you, the newcomer,
```

Comment:
Maybe add `You don't need to read or understand coala code base to solve a newcomer issue, but you do need to use git properly` after `very easy`. (iirc `git` would be emphasized later so it might not be necessary to mention git here.)